docs/src/components/mdx/index.tsx

```diff
@@ -15,7 +15,7 @@ import { Usage } from '../usage'
 import * as styles from './index.module.css'
 
 export const A: React.FC<AllHTMLAttributes<HTMLAnchorElement>> = props => {
-  const isExternal = props.href.includes('http')
+  const isExternal = /[a-z]+:\/\//.test(props.href)
```

Comment:
i feel like this line keep evolving and getting stronger. it's gaining sentience...

Reply:
Yes!  The next level is when I paste in something from the highest-voted StackOverflow answer.